core/App/ehsm_provider.cpp

```diff
@@ -1013,6 +1029,17 @@ ehsm_status_t VerifyQuote(ehsm_data_t *quote,
     //set current time. This is only for sample use, please use trusted time in product.
     current_time = time(NULL);
 
+    // check mr_signer and mr_enclave
+    if((mr_signer != NULL &&  strncmp(mr_signer, " ", strlen(mr_signer)) != 0) || 
+       (mr_enclave != NULL && strncmp(mr_enclave, " ", strlen(mr_enclave)) != 0)) {
+        ret = enclave_verify_policy(g_enclave_id, &sgxStatus, quote->data, quote->datalen, mr_signer, strlen(mr_signer), mr_enclave, strlen(mr_enclave));
```

Comment:
follow the line#1044 to format the function parameters.

Reply:
Ok, I will format it.